LibGAP.jl/test/conversion.jl

```diff
@@ -104,6 +104,15 @@
     xx = GAP.julia_to_gap( "a" )
     @test_throws MethodError GAP.gap_to_julia( Dict{Int64,Int64}, xx )
 
+    ## Test nothing
```

Comment:
While this is a cute joke, it's not really helpful...
```suggestion
    ## Test converting GAP lists with holes in them
```